test/e2e/flask/multichain-api/wallet_createSession.spec.ts

```diff
@@ -369,7 +369,7 @@ describe('Multichain API', function () {
   });
 
   describe('Dapp has existing session with 3 scopes and 2 accounts and then calls `wallet_createSession` with different scopes and accounts', function () {
-    const OLD_SCOPES = ['eip155:1337', 'eip155:1', 'eip155:42161'];
+    const OLD_SCOPES = ['eip155:1337', 'eip155:1'];
```

Comment:
Any idea why this was needed? i.e. was the test wrong in a way that didn't matter until now?

If we want to bring this in, we should update the description as well

Reply:
hmmm I thought this was related to the base mainnet issue, but now looking at it I don't think it is. Will revert the change here